examples/TargetingConsoleApp/Program.cs

```diff
@@ -63,11 +65,23 @@ public static async Task Main(string[] args)
                         Groups = user.Groups
                     };
 
+                    //
+                    // Evaluate feature flag using targeting
                     bool enabled = await featureManager.IsEnabledAsync(FeatureName, targetingContext, CancellationToken.None);
 
+                    //
+                    // Retrieve feature variant using targeting
+                    CartOptions cartOptions = await variantManager
+                        .GetVariantAsync<CartOptions, TargetingContext>(
+                            "ShoppingCart",
```

Comment:
For consistency, should we define this as a const strng as well, eg., DynamicFeatureName?

Reply:
Sounds good. Added